vmm/Cargo.toml

```diff
@@ -27,3 +27,11 @@ features = ["elf", "bzimage"]
 [dependencies.vm-memory]
 git = "https://github.com/rust-vmm/vm-memory"
 features = ["backend-mmap"]
+
+[dependencies.lazy_static]
+version = "1.0"
+optional = true
+
+[features]
+default = ["stable"]
+stable = ["lazy_static"]
```

Comment:
I think you should remove this whole declaration and instead add to the existing list of dependencies:
```
[dependencies]
...
kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "master" }
lazy_static = "1.0"
libc = ">=0.2.39"
...
```
The reason is, I don't see why we need to make this dependency optional.